packages/@sanity/vision/src/util/tryParseParams.js

```diff
@@ -1,8 +1,10 @@
+const JSON5 = require('json5')
+
 const tryParseParams = (val) => {
   try {
-    return val ? JSON.parse(val) : {}
+    return val ? JSON5.parse(val) : {}
   } catch (err) {
-    err.message = `Parameters are not valid JSON:\n\n${err.message}`
+    err.message = `Parameters are not valid JSON:\n\n${err.message.replace('JSON5:', '')}`
```

Comment:
Maybe add a comment about why we're replacing JSON5 from the original message? It wasn't all that clear to me why we needed it when reading this line.

Reply:
Will do (it was removed to clean up the error message, since it is always prefixed with `JSON5:`).